### PR TITLE
feat: `enableTracking` makes it possible to disable tracking

### PR DESCRIPTION
**Pull request by roncohen**
In situations where you do not want to send events or track user/company details on behalf of a user but still want to use them for feature targeting, you should be able to disable it. This is useful for example when and admin is impersonating another user.

### CODE REVIEW

**Review by pavkam — APPROVED** [review]
Look good

**Review by Swiftwork — APPROVED** [review]

File: packages/react-sdk/README.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ import { BucketProvider } from "@bucketco/react-sdk";
   <BucketProvider>
   ```
 
+- `impersonating` (default: `false`): Disables sending tracking events and user/company updates to Bucket and disabled automated feedback surveys. Set to `true` when impersonating another user.
```
Comment (matus-vacula) [inline packages/react-sdk/README.md:83]: The wording should be "enables".

Comment (roncohen) [reply packages/react-sdk/README.md:83]: OK i can see how this is confusing. If you set it to something other than the default then it'll disable sending tracking events .... Needs a rewording.

Comment (roncohen) [reply packages/react-sdk/README.md:83]: OK let me know if this is better.

**Review by pavkam — APPROVED** [review]

**Pull request merged**